### DISCUSSION TITLE
MySpeed - WebUI won't load/times out.

### DISCUSSION THREAD

**Discussion by SubNoizey** [tteck/Proxmox#3243] [Help Needed]
Not sure If I'm doing something wrong but I can't access the webUI of MySpeed via the script, I also tried installing it from the MySpeed Docs on a VM and I get the same outcome where the URL just times out. Another thing that I've found slightly odd is that MySpeed is giving me my external public IP and not my internal one. Maybe it's not listening on 192.168.1.101:5216 and only listening on my public IP which I'm not exposing to the internet.

**Comment by psilantropy** [comment]
Same problem here.

**Reply by tteck** [reply]
No issue here

![Screenshot 2024-07-23 7 58 53 AM](https://github.com/user-attachments/assets/51885aff-4925-45f9-aead-15f8b08d89d2)

**Reply by psilantropy** [reply]
Hi tteck :)

I ran this script again (3rd time), but disabled IPv6 on this run and it now works.
Reran again with ipv6 enabled and ERR CONNECTION REFUSED.

IPV6 shows as not connected in verbose logs, which is normal and expected.

IPV6 enabled;
![ipv6](https://github.com/user-attachments/assets/99d3e727-41b6-4d32-bb30-79f9a2561342)

IPV6 disabled;
![ipv6_dis](https://github.com/user-attachments/assets/a7dd5912-a26b-4284-a276-65c941eadaee)
Disabled and working also runs 3 more processes looking at top. Command node, sd-pam, and something else.

If that doesn't help, let me know if you want the full verbose logs. Happy to redeploy and assist.

**Reply by SubNoizey** [reply]
> Hi tteck :)
> 
> I ran this script again (3rd time), but disabled IPv6 on this run and it now works. Reran again with ipv6 enabled and ERR CONNECTION REFUSED.
> 
> IPV6 shows as not connected in verbose logs, which is normal and expected.
> 
> IPV6 enabled; ![ipv6](https://private-user-images.githubusercontent.com/79955883/351500342-99d3e727-41b6-4d32-bb30-79f9a2561342.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQzMzEyMDEsIm5iZiI6MTcyNDMzMDkwMSwicGF0aCI6Ii83OTk1NTg4My8zNTE1MDAzNDItOTlkM2U3MjctNDFiNi00ZDMyLWJiMzAtNzlmOWEyNTYxMzQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIyVDEyNDgyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI0ZDUyNWI3MWUyODdlNmYzZjc2NWQyZmZiYjhlNDQ5MTZiNTgwMTc3M2EyOWM5ODUxODk0NTkwMWE3N2E1YzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._qy43-vOeKuf9e8SoZDETKWCGi8HZlyeW9i9vN5oPHs)
> 
> IPV6 disabled; ![ipv6_dis](https://private-user-images.githubusercontent.com/79955883/351500500-a7dd5912-a26b-4284-a276-65c941eadaee.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQzMzEyMDEsIm5iZiI6MTcyNDMzMDkwMSwicGF0aCI6Ii83OTk1NTg4My8zNTE1MDA1MDAtYTdkZDU5MTItYTI2Yi00Mjg0LWEyNzYtNjVjOTQxZWFkYWVlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIyVDEyNDgyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIwOGEzMTc5M2Y0OGMzNzEyZTcxYTRjMTRhZGQ0MWEyMDk5MTIwNmZhNTdiZWIzMDQ3MTJiNTlkZTM1MmFlZmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.fxbRm-6iiJVJ0oi93Go0ZegYiDjcxEIeIcdgzl6eBIU) Disabled and working also runs 3 more processes looking at top. Command node, sd-pam, and something else.
> 
> If that doesn't help, let me know if you want the full verbose logs. Happy to redeploy and assist.

Brilliant, did an advanced setup and disabled IPv6 and it works.
